abi/vfabia64/vfabia64.rst

```diff
@@ -352,27 +356,24 @@ Change history
    |           |               |   parameters from ``decimal`` to ``integrals``,  |
    |           |               |   and improve the description of the rules.      |
    +-----------+---------------+--------------------------------------------------+
-   |2019Q4alpha|31st October   |Github preview release with an open source        |
-   |           |2019           |license.                                          |
+   |2019Q4     |30th January   |Github preview release with an open source        |
+   |           |2020           |license.                                          |
    |           |               |                                                  |
-   |           |               |Major changes are:                                |
+   |           |               |Major changes:                                    |
```

Comment:
remove trailing whitespaces

Reply:
it's a feature, the text is in a table :)

Reply:
Facepalm. Apologies for wasting your time.